app/templates/index.html

```diff
@@ -46,6 +48,205 @@ <h4>LET’S CONNECT</h4>
+                            </div> 
+                        </div>
+                        <div class="year">2015</div>
+                    </div>
+                    <div class="section-primary-text">
+                        <p>
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="section-tags">
+                        <button type="submit">html</button>
+                        <button type="submit">css</button>
+                        <button type="submit">javascript</button>
+                    </div>
+                    <div class="section-action">
+                        <button type="submit">See project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        
+        <!--Fourth middle section-->
+        <section class="section-elements">
+            <div class="middle-section">
+                <div class="section-four-bg"></div>
+                <div class="section-block">
+                    <div class="section-title"><h2>Multi-Post Stories</h2></div>
+                    <div class="section-frame">
+                        <div class="cannopy">CANOPY
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div>
+                        </div>
+                        <div class="backend">Back End Dev
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div> 
+                        </div>
+                        <div class="year">2015</div>
+                    </div>
+                    <div class="section-primary-text">
+                        <p>
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="section-tags">
+                        <button type="submit">html</button>
+                        <button type="submit">css</button>
+                        <button type="submit">javascript</button>
+                    </div>
+                    <div class="section-action">
+                        <button type="submit">See project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+    
+        <!--About me section-->
+        <section class="section-elements">
+            <div class="section-about">
+                <div class="section-block about">
+                    <div class="about-me-text">
+                        <h1>About <br> Myself</h1>
```

Comment:
- [ ] While you have made a great effort to use a header tag, kindly note that for ```<h1>``` it is usually a special case. There needs to be only one ```<h1>``` per page mainly because of syntactical and semantical reasons. It would be better to instead use a ```<h2>``` for this part instead.